### ISSUE TITLE
sample compilation error for esp32

### ISSUE THREAD

**Issue by robademar** [Aircoookie/Espalexa#40]
First of all thanks to the author(s) of this code: it seems simple to use and even works!
I tested the sample code with esp8266 and it works with the combination of "v2 higher bandwidth" and esp board 2.5.0 beta 2, while with beta 3 I've got a compilation error.
So for esp8266 no problem.
I then tried with an esp32 and I've got this error:
C:\Users\roberto\Documents\arduino librerie\libraries\Espalexa-master\src/Espalexa.h:40:94: fatal error: WebServer.h: No such file or directory

compilation terminated.

Any idea?
thanks again

**Comment by Aircoookie** [comment]
Hi, you are probably running an old version of the ESP32 Arduino core.
Please upgrade to 1.0.x, only that contains the missing WebServer library.
If you don't like that, you may also download AsyncTCP and ESPAsyncWebserver and add `#define ESPALEXA_ASYNC` before the Espalexa include to use the library in async mode.

**Comment by robademar** [comment]
I've updated esp32 arduino core and now it is correctly compiled.
But as opposite to the esp8266, discovery doesn't find any device.
I tried changing cpu frequency to 240Mhz, before at lower speed the board was continuously resetting, but noway to fidìnd the devices.
I added debug and this is the log:
rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:1
load:0x3fff0018,len:4
load:0x3fff001c,len:1100
load:0x40078000,len:10088
load:0x40080400,len:6380
entry 0x400806a4
[D][WiFiGeneric.cpp:342] _eventCallback(): Event: 0 - WIFI_READY
[D][WiFiGeneric.cpp:342] _eventCallback(): Event: 2 - STA_START

Connecting to WiFi
Connecting.....[D][WiFiGeneric.cpp:342] _eventCallback(): Event: 4 - STA_CONNECTED
[D][WiFiGeneric.cpp:342] _eventCallback(): Event: 7 - STA_GOT_IP
[D][WiFiGeneric.cpp:385] _eventCallback(): STA IP: 192.168.1.6, MASK: 255.255.255.0, GW: 192.168.1.1
.
Connected to Vodafone-roberto
IP address: 192.168.1.6
Constructing device 1
Adding device 1
Constructing device 2
Adding device 2
Adding device 3
Espalexa Begin...
MAXDEVICES 10
Done
Got UDP!
NOTIFY * HTTP/1.1
HOST: 239.255.255.250:1900
CACHE-CONTROL: max-age=100
LOCATION: http://192.168.1.250:49152/description.xml
NT: urn:schemas-wifialliance-org:service:WFAWLANConfig:1
NTS: ssdp:alive
SERVER: Linux/2.6.36, UPnP/1.0, Portable SDK for U


any suggestion?
thanks
